### PR TITLE
rename some stuff, toggle input display from debug menu, check in .dll

### PR DESCRIPTION
**Pull request by dallmeyer**

### CODE REVIEW

File: goal_src/jak1/game.gp [hunk 1]
```diff
@@ -2079,16 +2079,22 @@
...
+ "mods/mod-settings.gc"
+ "mods/mod-utils.gc"
+)
+
+(goal-src-sequence
+ ;; prefix
+ "engine/"
+ :deps ("$OUT/obj/mod-settings.o" "$OUT/obj/mod-utils.o" "$OUT/obj/battlecontroller.o" "$OUT/obj/snow-bunny.o" "$OUT/obj/baby-spider.o" "$OUT/obj/sage-village3.o" "$OUT/obj/sage-finalboss.o" "$OUT/obj/assistant-citadel.o" "$OUT/obj/assistant-lavatube.o" "$OUT/obj/robocave-part.o" "$OUT/obj/driller-lurker.o" "$OUT/obj/training-part.o" "$OUT/obj/rolling-race-ring.o" "$OUT/obj/beach-part.o" "$OUT/obj/sculptor.o" "$OUT/obj/sunken-fish.o" "$OUT/obj/billy.o" "$OUT/obj/sidekick-human.o" "$OUT/obj/flying-lurker.o" "$OUT/obj/target-racer-h.o" "$OUT/obj/firecanyon-obs.o" "$OUT/obj/target-flut.o" "$OUT/obj/hud-classes-pc.o" "$OUT/obj/collide-reaction-racer.o" "$OUT/obj/plant-boss.o" "$OUT/obj/beach-obs.o" "$OUT/obj/sunken-elevator.o" "$OUT/obj/jungle-part.o" "$OUT/obj/sequence-a-village1.o" "$OUT/obj/ticky.o")
+ "mods/mod-custom-code.gc"
 )
```
Comment (Zedb0T) [inline goal_src/jak1/game.gp:2098]: Why is this separated into two now?

File: goal_src/jak1/engine/mods/input-display.gc [hunk 2]
```diff
@@ -321,50 +321,56 @@ Down lets you edit buttons, left lets you edit the sticks, right for dpad, up to
...
+    ;process-spawn-function, spawns a process that runs the function you give it
+    (process-spawn-function process :name 'button-proc
+      ;This lambda is our function
+      (lambda :behavior process ()
+        ;Code before the loop runs once on process spawn
+        ;
+        ;Loop runs once per frame while process is active
+        (loop
           
-          (draw-stick2 testbuf)
```
Comment (Zedb0T) [inline goal_src/jak1/engine/mods/input-display.gc:354]: We dont like drawing both sticks?

Comment (dallmeyer) [reply goal_src/jak1/engine/mods/input-display.gc:354]: I didn't change this 🤷

File: goal_src/jak1/game.gp (hunk 1 above)
Comment (dallmeyer) [reply goal_src/jak1/game.gp:2098]: will revert

**Pull request merged**